### PR TITLE
tests: fix upgrade-from-release test in ubuntu noble

### PR DESCRIPTION
**Pull request by sergiocazzolato**
update the test to include the proper snapd version and skip it in ubuntu  24.10

### CODE REVIEW

File: tests/main/upgrade-from-release/task.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ restore: |
     distro_install_build_snapd
 
 execute: |
-    if os.query is-ubuntu 24.04; then
+    if os.query is-ubuntu 24.10; then
         echo "Ubuntu noble is skipped until it is released."
```
Comment (olivercalder) [inline tests/main/upgrade-from-release/task.yaml:21]: s/noble/oracular/

:)

**Review by olivercalder — COMMENTED** [review]
Looks good, one comment.

**Review by Meulengracht — APPROVED** [review]

**Review by olivercalder — APPROVED** [review]
Thanks for the change!

**Review by andrewphelpsj — APPROVED** [review]
thanks!

**Pull request merged**